packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -209,7 +209,7 @@ const AccountTransferForm = ({
             </h2>
             <Formik
                 initialValues={{
-                    amount: account_transfer_amount,
+                    amount: '',
```

Comment:
why are you removing this? we need this so if you come back from confirmation screen we can populate the latest fields you set up

Reply:
on first page load `account_transfer_amount` is undefined which makes `amount` an uncontrolled field, yet after key in values it will become controlled again and throws error.

Reply:
workaround is that to set it to ' ' if `account_transfer_amount` is undefined. I will update the code.